### ISSUE TITLE
Invoking File Store Service - Regd

### ISSUE THREAD

**Issue by balajijana** [ptkach/fileDroppa#48]
Hi 

When I have invoked the file store service i was getting the below error:

Uncaught SyntaxError: Unexpected token <
	Evaluating http://localhost:3001/node_modules/file-droppa/index/components/Services/FileStore.service.js

I have invoked like this:

import {FilesStore} from 'file-droppa/components/Services/FileStore.service';

@Component({  
  providers: [RouterLink,JwtHelper,IdentityService,AuthService, FilesStore]
})

constructor(@Inject(FormBuilder) public formBuilder:FormBuilder,
              @Inject(Router) public router:Router,
              @Inject(IdentityService) public IdentityService:IdentityService,
              @Inject(AuthService) public authService:AuthService,
              @Inject(JwtHelper) public jwthelper: JwtHelper,
              @Inject(FilesStore) public filesStore:FilesStore)

Please let me know how to resolve this error

**Comment by ptkach** [comment]
Why are you invocing FileStore service? It's internal FileDropZone helper, do you see any word about it in Readme? 
**Stop opening issues like that!** Everyday you open some issue, I answer you question and close issue and no feedback from you. Are u opening issues just to open issues? GitHub Activity?

**Comment by balajijana** [comment]
Hi ptkach,
  Actually I have seen that in the following url: https://github.com/ptkach/fileDroppa/issues/38
I was actually implementing in this way: 
fileUploaded([success, response, file]){
        success && console.log("uploaded - awesome", response, file);
        success || console.log("not uploaded - very bad", response, file);
    }

    filesUpdated(files) {
        console.log("added", files)
    }
In the fileuploaded() i am not getting any values in the response or file parameter. So that only I have used the Filestore. I have checked if(success){console.log(response,file)}, but not value in the console.
